src/components/Results.tsx

```diff
@@ -23,20 +23,26 @@ const defaultCenter: LatLngTuple = [51.505, -0.09];
 export const Results: React.FC<{
   center: LatLngTuple;
 }> = ({ center }) => {
-  const { facilities, searchParams } = useAppState();
+  const { facilities, searchParams, selectedFacilityId } = useAppState();
   const dispatch = useAppDispatch();
   const { winWidth } = useWindowsDimension();
 
   const [facilityIds, setFacilityIds] = useState<string[]>([]);
   const [loading, setLoading] = useState<boolean>(false);
   const [error, setError] = useState<undefined | string>();
-  const [selectedFaciltyId, setSelectedFacilityId] = useState<undefined | string>();
 
   const filteredFacilities = useMemo(
     () => facilities.filter((f) => facilityIds.includes(f.id)),
     [facilities, facilityIds]
   );
 
+  useEffect(() => {
+    dispatch({
+      type: "RESET_SELECTED_FACILITY_ID",
+      payload: undefined
+    })
+  }, []);
```

Comment:
`dispatch` must be added to the dependencies of `useEffect`

Reply:
btw, we used to have a warnings on this type off errors, but from some point they stopped appearing